src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewTextViewCell.swift

```diff
@@ -0,0 +1,91 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import Foundation
+import UIKit
+
+class DynamicTableViewTextViewCell: UITableViewCell, DynamicTableViewTextCell {
+	private let textView = UITextView()
+
+	var dataDetectorTypes: UIDataDetectorTypes {
+		get { textView.dataDetectorTypes }
+		set { textView.dataDetectorTypes = newValue }
+	}
```

Comment:
This should be exposed as an additional configure method `configureTextView(dataDetectorTypes: ...)`